Makefile

```diff
@@ -5,6 +5,12 @@ ifeq (${PREFIX},)
 	PREFIX := /usr/local
 endif
 
+GOBIN ?= $(shell go env GOPATH)/bin
+
+GOIMPORTSREVISER = $(GOBIN)/goimports-reviser
+$(GOIMPORTSREVISER):
+	go install -v github.com/incu6us/goimports-reviser/v3@latest
```

Comment:
As discussed, let's remove auto installation for now and make an issue to cover all util bins we depend on at once